core/src/main/java/org/carbondata/core/util/DataTypeUtil.java

```diff
@@ -1,3 +1,22 @@
+/*
```

Comment:
It seems the latest code of `org.carbondata.scan.util.DataTypeUtil` is not merged properly. Please take the latest code of `DataTypeUtil` and merge it.